test.dockerfile

```diff
@@ -35,7 +35,8 @@ RUN pip install tox
 # Install the relevant dependencies. Keep printing so Travis knows we're alive.
 RUN ["bash", "-c", "( while true; do echo '.'; sleep 60; done ) & tox --notest"]
 
-# Finally, upload our actual code!
+# Finally, cleans cached code and upload our actual code!
+RUN rm -rf ./gym*
```

Comment:
Sure, that'll work.

Maybe nicer to move to `/tmp`, but minor detail. (I also prefer to use `.tox` / `rm -rf .`, but minor.)